cts/cts-cli.in

```diff
@@ -3354,6 +3370,7 @@ for t in $tests; do
         -e 's/last_change time=\".*\"/last_change time=\"\"/' \
         -e 's/ api-version="[^"]*"/ api-version="X"/' \
         -e 's/ default="[^"]*"/ default=""/' \
+        -e 's/ Default value: .*/ Default value:/' \
```

Comment:
hmm, no great way around it ... maybe `s/ Default value: (val1|val2)/ Default value: OPT1_DEFAULT/` for known ones ...

Reply:
That's going to grow the sed significantly, and I'm not sure how much of a disadvantage there is to leaving it this way... a bit aggressive, but it'll affect only text output for whatever tools it hits.

Either way, the sed lines will have to be tweaked again depending on the text output format we settle on.

Reply:
Enumerating known ones feels like asking for trouble if they change in the future -- for example, if we add a new allowed value and a configure option to choose it as the default. We won't catch that `cts-cli` fails with that setting unless one of our CI machines uses that configure option.

Reply:
What I meant was enumerating just the values that *can* be configured differently. The idea being

    concurrent-fencing ...
        ...
        Allowed values: boolean (default: false)

or

    concurrent-fencing ...
        ...
        Allowed values: boolean (default: true)

would be replaced with

    concurrent-fencing ...
        ...
        Allowed values: boolean (default: CONCURRENT_FENCING_DEFAULT)

where CONCURRENT_FENCING_DEFAULT is a literal string and not a placeholder. But, we won't have the option name easily available, so probably just stripping defaults is the way to go.